index.js

```diff
@@ -0,0 +1,12 @@
+const hamburger = document.querySelector('.hamburgur');
+const navsub = document.querySelector('.nav-sub');
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('change');
+  navsub.classList.toggle('nav-change');
+});
+
+document.querySelectorAll('.nav-link').forEach((n) => n
+  .addEventListener('click', () => {
+    hamburger.classList.remove('change');
+    navsub.classList.remove('nav-change');
+  }));
```

Comment:
- [ ] It is required to stick to the Figma design as close as possible. T

Reply:
Hello @Qoosim kindly where is the issue? I have not understood, because I used the figma design, is it the pop up menu?